cob_bringup/tools/teleop_v1.xml

```diff
@@ -10,8 +12,10 @@
 	</include>
 
 	<node pkg="cob_teleop" type="cob_teleop_v1" name="$(anon teleop_node)" ns="teleop" output="screen">
-		<remap from="/cmd_vel" to="/base_controller/command" />
+		<remap from="/cmd_vel" to="$(arg cmd_vel)" />
 		<remap from="/cmd_vel_safety" to="/base_controller/command_safe" />
+		<remap from="/base_controller/init" to="$(arg service_ns)/init" />
+		<remap from="/base_controller/recover" to="$(arg service_ns)/recover" />
 		<rosparam command="load" ns="/robot_config" file="$(arg pkg_hardware_config)/$(arg robot)/config/robot_modules.yaml"/>
 		<rosparam command="load" ns="/teleop" file="$(arg pkg_hardware_config)/$(arg robot)/config/teleop.yaml"/>
```

Comment:
order: ns, command file

why is there always a ```/``` at the beginning...?

btw. should we put the whole thing into a <group ns="teleop"></group>?